includes/scheduler/class-post.php

```diff
@@ -149,4 +156,35 @@ public static function send_announces( $outbox_activity_id, $activity_object, $a
 		// Schedule the outbox item for federation.
 		Scheduler::schedule_outbox_activity_for_federation( $outbox_activity_id );
 	}
+
+	/**
+	 * Filter the post data before it is inserted via the REST API.
+	 *
+	 * @param \WP_Post         $post     The post object.
+	 * @param \WP_REST_Request $request  The request object.
+	 *
+	 * @return \WP_Post The post object.
```

Comment:
```suggestion
	 * @param \stdClass        $post     An object representing a single post prepared for inserting or updating the database.
	 * @param \WP_REST_Request $request  The request object.
	 *
	 * @return \stdClass The prepared post.
```
Not sure [if it matters](https://developer.wordpress.org/reference/hooks/rest_pre_insert_this-post_type/#parameters)